sqlglot/expressions.py

```diff
@@ -4220,6 +4246,51 @@ def _replace_placeholders(node, args, **kwargs):
     return expression.transform(_replace_placeholders, iter(args), **kwargs)
 
 
+def func(name: str, *args, **kwargs) -> Func:
+    """
+    Returns a Func expression.
+
+    Examples:
+        >>> abs = func("abs", this=Literal.number(5))
+        >>> abs.sql()
+        'ABS(5)'
+        >>>
+        >>> cast = func("cast", this=Literal.number(5), to=DataType.build("DOUBLE"))
+        >>> cast.sql()
+        'CAST(5 AS DOUBLE)'
+
+    Args:
+        name: the name of the function to build.
+        args: the args used to instantiate the function of interest.
+        kwargs: the kwargs used to instantiate the function of interest.
+
+    Note:
+        The arguments `args` and `kwargs` are mutually exclusive.
+
+    Returns:
+        An instance of the function of interest, or an anonymous function, if `name` doesn't
+        correspond to an existing `sqlglot.expressions.Func` class.
+    """
+    if args and kwargs:
+        raise ValueError("Can't use both args and kwargs to instantiate a function.")
+
+    from sqlglot.parser import Parser
+
+    from_args_list = Parser.FUNCTIONS.get(camel_to_snake_case(name))
```

Comment:
why do you need to call camel_to_snake_case?

Reply:
Because of line 2711:

```python
cls._sql_names = [camel_to_snake_case(cls.__name__)]
```

If we're relying on `Parser.FUNCTIONS` to find a matching `exp.Func` class, I think we need this as a pre-processing step for `name`.

Reply:
i don't get it? the user is passing in the snake case already, we don't need to convert that

Reply:
Ah yeah, I see what you mean. Yeah, it doesn't really make sense. I'm reverting this, thanks.

Reply:
Fixed